src/tests/unit/tests/DetailsView/components/tab-stops/tab-stops-requirements-table.test.tsx

```diff
@@ -54,28 +67,37 @@ describe('TabStopsRequirementsTable', () => {
     });
 
     test('renders table', () => {
-        const testSubject = shallow(<TabStopsRequirementsTable {...props} />);
-        expect(testSubject.getElement()).toMatchSnapshot();
+        const testSubject = render(<TabStopsRequirementsTable {...props} />);
+        expect(testSubject.asFragment()).toMatchSnapshot();
     });
 
     test('renders requirement column', () => {
-        const testSubject = shallow(<TabStopsRequirementsTable {...props} />);
-        const columns = testSubject.find(DetailsList).props().columns;
-        expect(columns[0].onRender(requirementContentStub)).toMatchSnapshot();
+        render(<TabStopsRequirementsTable {...props} />);
+        const hasColumns = getMockComponentClassPropsForCall(DetailsList).columns;
+        expect(hasColumns[0].onRender(requirementContentStub)).toMatchSnapshot();
+        expect(hasColumns).toMatchSnapshot();
     });
 
     test('renders result column', () => {
-        const testSubject = shallow(<TabStopsRequirementsTable {...props} />);
-        const columns = testSubject.find(DetailsList).props().columns;
-        const tabStopsChoiceGroup = columns[1].onRender(requirementsList[0]);
+        const testSubject = render(<TabStopsRequirementsTable {...props} />);
+
+        const tabStopsChoiceGroup = testSubject.container.querySelectorAll('.ms-ChoiceFieldGroup');
         expect(tabStopsChoiceGroup).toMatchSnapshot();
     });
 
+    test('renders undo icon button', async () => {
```

Comment:
Where did this test come from?

Reply:
Hi 
This test case is not required as per latest changes, hence removed now, please re-review.